src/cloud/project/project-conf-files_services.md

```diff
@@ -43,19 +43,19 @@ elasticsearch:
 
 ## Service values
 
-You must provide the _type_ values: service _name_ and _version_. If the service uses persistent storage, then you must provide a _disk_ value. Use the following format:
+You must provide the _type_ values: _service-name_ and _version_. If the service uses persistent storage, then you must provide a _disk_ value. Use the following format:
```

Comment:
Why did we change this to `service-name` here and in the YAML example below but leave it as just `name` in the documentation (eg, the heading on line 54 and the following two paragraphs [56 and 58]).

I think `name` is fine, but either way we should be consistent.

Reply:
Yes, need to review all examples, but not we have 2 `<name>` in this schema https://devdocs.magento.com/cloud/project/project-conf-files_services.html#service-values what is wrong because service names are limited (redis, mysql, elasticsearch and rabbitmq), when service id can have any value with alphanumeric characters. 
Probably we may add this as
`<service-id>:
    type: <name>:<version>
    disk: <value-MB>`

Reply:
@meker12 can you please suggest better naming for fields to avoid `<name>` for two different fields

Reply:
service-id -- works. Added suggestions to PR.